app/assets/stylesheets/custom.scss

```diff
@@ -66,8 +66,23 @@ li {
   letter-spacing: -1px;
   padding-top: 9px;
   font-weight: bold;
+  padding-bottom: 9px;
   &:hover {
     color: #fff;
     text-decoration: none;
   }
+}
+
+#liCurrUser {
```

Comment:
We tend to use elements' classes to define styling, instead of elements' ids.
Secondly, dash notation is the most favourite to define CSS classes.